pkg/upstream/admin-console.go

```diff
@@ -32,6 +32,29 @@ type UpstreamSettings struct {
 	IncludeMinio           bool
 }
 
+func WriteAdminConsoleFiles(renderDir string, options types.WriteOptions) error {
```

Comment:
i don't think this function belongs in the upstream package IMO, i would just dump the functions logic in the cli command itself